packages/clay-css/src/scss/components/_buttons.scss

```diff
@@ -1,261 +1,147 @@
 .btn {
-	background-color: transparent;
-	border: $btn-border-width solid transparent;
-
-	@include border-radius($btn-border-radius, 0);
-	@include box-shadow($btn-box-shadow);
-
-	color: $body-color;
-	cursor: $btn-cursor;
-	display: inline-block;
-	font-family: $btn-font-family;
-	font-size: $btn-font-size;
-	font-weight: $btn-font-weight;
-	line-height: $btn-line-height;
-	padding: $btn-padding-y $btn-padding-x;
-	text-align: center;
-	text-transform: none;
+	background-color: var(--btn-bg, transparent);
+	border: var(--btn-border-width, 1px) var(--btn-border-style, solid)
+		var(--btn-border-color, transparent);
+	border-radius: var(--btn-border-radius, 0.25rem);
+	box-shadow: var(--btn-box-shadow);
+	color: var(--btn-color, inherit);
+	cursor: var(--btn-cursor, pointer);
+	display: var(--btn-display, inline-grid);
+	font-family: var(--btn-font-family);
+	font-size: var(--btn-font-size);
```

Comment:
using a custom property undefined we will set the component for future customizations